catalog/core/catalog-core-solr/pom.xml

```diff
@@ -202,7 +202,19 @@
         <dependency>
             <groupId>com.google.truth</groupId>
             <artifactId>truth</artifactId>
-            <version>0.39</version>
+            <version>0.42</version>
```

Comment:
✏️ Versions shouldn't be hardcoded at that level. Versions should be defined in the top pom.

Reply:
No other projects are using Truth yet.  It is being tested in this project.

Reply:
Nevertheless, This eventually leads to multiple projects using different versions. As part of a meeting we had last week, it was deemed that having versions defined in multiple places have been a source of problems and in order to avoid this, we should define all versions in the same location.